mule-apikit-module/src/main/java/org/mule/module/apikit/Router.java

```diff
@@ -126,7 +111,7 @@ public CoreEvent process(final CoreEvent event) throws MuleException {
       } catch (Exception e) {
         return Flux.error(buildMessagingException(e));
```

Comment:
Use `Mono.error` for consistency.  Also double check to make sure `buildMessagingException` is still needed (the one in the `try` primarily).

I don't see any use of child event contexts here, so not sure what was referred to previously?